BrianLusina/chat-sockets
# Settings Page Selectors and Options

## Description

Settings Page Selectors and Options
1. UserProfile to change username
2. ThemeSelector to change theme option

## How can we test this?

1. Run `yarn test` in root of client directory for unit tests
2. Run `yarn start` in root of client directory
3. Access Settings Page http://localhost:3000/#/settings and Change username,
4. Refresh page, the username you set should be reloaded
5. Change theme from RadioGroup option. When you refresh the page, this options should be pre-selected

## Checklist:
